core/templates/dev/head/editor/EditorServices.js

```diff
@@ -215,6 +241,18 @@ oppia.factory('changeListService', [
     }
     explorationChangeList.push(changeDict);
     undoneChangeStack = [];
+
+    // FIXME: Uncomment this block when the autosave PUT request url is fixed.
```

Comment:
I had left this as a reminder to later fix the url. This isn't intended to be shipped into the codebase.